README.md

```diff
@@ -1,22 +1,42 @@
-# cosmoz-autocomplete
-
 [![Build Status](https://travis-ci.org/Neovici/cosmoz-autocomplete.svg?branch=master)](https://travis-ci.org/Neovici/cosmoz-autocomplete)
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://www.webcomponents.org/element/owner/my-element)
+
+# &lt;cosmoz-autocomplete&gt;
+
+`cosmoz-autocomplete` is a Polymer Component to combine drop-down like behaviour with searchable filtering behaviour. 
+Supports both single and multiple selections.
 
-cosmoz-autocomplete is a Polymer component to combine drop-down like behaviour with searchable filtering behaviour. Supports both single and multiple selections.
+## Installation
 
-## Credits
+To install, run: `bower install --save Neovici/cosmoz-autocomplete`
 
 ## Usage
 
-### Install
+For example, we have the following `data` array:
+```js
+data = ["Red", "Green", "Blue", "White", "Black"]
+```
 
-`bower install --save Neovici/cosmoz-autocomplete`
+And an array for storing selected items:
+```js
+selected = []
+```
 
-### Add the cosmoz-autocomplete import
+The following code will create `cosmoz-autocomplete` with multiple selection feature:
```

Comment:
Inline-demo is not a mandatory. I think simple example is not worse. Earlier I faced with some issues when using data binding in inline demo, therefore I used static example.